blazar/plugins/oshosts/host_plugin.py

```diff
@@ -515,6 +520,24 @@ def delete_computehost(self, host_id):
             # they have to rerun
             raise manager_ex.CantDeleteHost(host=host_id, msg=str(e))
 
+    def handle_disabled_key(self, host_id, value):
+        # only admin can set/unset 'disabled' flag
+        if self._is_admin():
+            host_details = self.get_computehost(host_id)
+            new_disabled_flag = False if value is None else True
```

Comment:
Should this not be `False if value is None else value`?

Reply:
The `value` is parsed as a string. As there is no schema or exact string the user can use (they can use 'true', 'True', 'T', or any string for that matter). The only thing matters is `set` or `unset` while the set needs a keykey-value pair, but unset needs only the key with value of `None`. So as long as `set` is called with some _value_ this will work. 

I am not sure about the proper way this can be documented.

Reply:
Can you share the commands you were using?